### DISCUSSION TITLE
Add Update Branch button for PRs

### DISCUSSION THREAD

**Discussion by rambleraptor** [community/community#16348] [Mobile] [answered]
Being able to merge PRs on mobile is so great, especially when you have long-running checks. 

It would be great to be able to update the branch on a PR on mobile. Paired with turning auto merge on/off, this lets you handle every form of merging a PR on mobile. 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.64.0 (66902451)
Device: iPhone 13 Pro (iOS 15.4.1)
</details>

**Comment by amrehman** [comment, marked as answer]
Hi there @rambleraptor ! Thanks a lot for this feedback. 👍🏽

I am going to create a tracking issue for this on our side! Stay tuned to our future releases! 🚀
